cmd/relay/relay_internal_test.go

```diff
@@ -72,3 +78,60 @@ func TestRunBootnodeAutoP2P(t *testing.T) {
 	testutil.SkipIfBindErr(t, err)
 	require.NoError(t, err)
 }
+
+func TestServeMultiAddrs(t *testing.T) {
```

Comment:
can we have `testutil.RequireGoldenJSON(...)` for this test for json response of http call?